db/migrate/20180129223616_add_group_to_team_members.rb

```diff
@@ -0,0 +1,7 @@
+class AddGroupToTeamMembers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :team_members, :description, :text
+    add_column :team_members, :group, :string, null: "false"
+    add_column :team_members, :image_src, :string, default: "images/img_unk.png"
```

Comment:
Let's not set a default here for the time being, and allow for scoping this by `image_src: nil`.